pkgs/tools/admin/ejson2env/default.nix

```
@@ -20,6 +20,10 @@ buildGoModule rec {
   ];
 
   passthru.updateScript = nix-update-script { };
+  passthru.tests = {
```

Not blocking, but since we are here...

```suggestion
  passthru = {
    updateScript = nix-update-script { };
    tests = {
```